cluster/charts/crossplane/templates/rbac-manager-deployment.yaml

```diff
@@ -74,9 +74,6 @@ spec:
         args:
         - rbac
         - start
-        {{- if .Values.rbacManager.managementPolicy }}
-        - --manage={{ .Values.rbacManager.managementPolicy }}
```

Comment:
this way there would be no way to set the deprecated flag through the helm chart, so people really wanting to use that would need to either fork the chart, install it some other way or patch it downstream. We could probably rename the field in the values to a `deprecatedManagementPolicy`, leave the old `managementPolicy` as deprecated and [error out](https://helm.sh/docs/chart_template_guide/function_list/#fail) directly from helm too with a similar message in case a user set it.

Reply:
I was thinking you could still set it using `rbacManager.args` (see line below) if you needed to.